src/deserialize/parse.rs

```diff
@@ -197,34 +197,79 @@ fn read_discard(chars: &mut std::iter::Enumerate<std::str::Chars>) -> Result<Opt
 }
 
 fn read_number(n: char, chars: &mut std::iter::Enumerate<std::str::Chars>) -> Result<Edn, Error> {
+    let delimiters = [',', ']', '}', ')', ';'];
```

Comment:
This could be a constant, it would prevent reallocatiion for every time this function is called.

Reply:
repeat of  https://github.com/naomijub/edn-rs/pull/101#discussion_r1306860246
will revisit after that's resolved, as that'll probably be first.

Reply:
actually, a rebase would fix this